tests/stress/ebpf_mt_stress.h

```diff
@@ -53,8 +53,8 @@ struct test_control_info
     // The number of threads allocated per jit program.
     uint32_t threads_count{0};
 
-    // The run time for each jit program test thread in seconds.
-    uint32_t duration{0};
+    // The run time for each jit program test thread in minutes.
+    uint32_t duration_min{0};
```

Comment:
Coding conventions are to avoid abbreviations. Acronyms like "ms" are ok but not "min", which could be "minimum" or "minutes".

Reply:
fixed